Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -2470,20 +2471,26 @@ scheduledRideExistsPopUpConfig state =
   formatDateInHHMM timeUTC = EHC.convertUTCtoISC timeUTC "HH" <> ":" <> EHC.convertUTCtoISC timeUTC "mm"
 
 
-getPickupMarkerActionImageConifg :: Boolean -> Boolean -> JB.ActionImageConfig 
-getPickupMarkerActionImageConifg pickupFeatureEnabled driverWithinPickupThreshold = 
-  if ((any (\stage -> isLocalStageOn stage) [ RideAccepted, ChatWithDriver]) && pickupFeatureEnabled && driverWithinPickupThreshold) then 
+getDestMarkerActionImageConifg :: ST.HomeScreenState -> Boolean -> JB.ActionImageConfig 
```

Comment:
Can we give proper name to this function as it's used for both pickup and dest marker ?

Reply:
DestMArker is used because in both the case for route it is the destination marker where we have to add image with label.